src/ApiService/ApiService/onefuzzlib/Config.cs

```diff
@@ -0,0 +1,66 @@
+﻿using ApiService.OneFuzzLib.Orm;
+using System;
+using System.Threading.Tasks;
```

Comment:
Can we rename this to InstanceConfig.cs? Config.cs feels a little too generic.

Reply:
Was just using 1 to 1 with Python code. 

Renamed to InstanceConfig.cs